File: 백준/Bronze/2525. 오븐 시계/오븐 시계.js
var fs = require('fs');
var input = fs.readFileSync('/dev/stdin').toString().split('\n');
var time = input[0].split(' ');
var hour = parseInt(time[0]);
var minute = parseInt(time[1]);
var cookTime = parseInt(input[1]);

minute += cookTime;
if(minute >= 60) {
    hour += Math.floor(minute / 60);
    minute %= 60;
}
if(hour >= 24) hour %= 24;

console.log(hour + " " + minute);

File: 백준/Bronze/2525. 오븐 시계/오븐 시계.py
H, M = map(int, input().split())
N = int(input())

# 총 분을 계산한다.
total_minutes = H * 60 + M + N

# 총 분을 다시 시간과 분으로 변환한다.
new_H = (total_minutes // 60) % 24  # 24시간을 넘기면 0으로 리셋해야 하므로 % 24를 사용

new_M = total_minutes % 60

print(new_H, new_M)
